scripts/setup_roboschool.sh

```diff
@@ -0,0 +1,37 @@
+echo "Make sure this script is running under garage conda environment"
+
+# install dependency
+if [[ "$(uname)" == "Darwin" ]]; then
+	brew install cmake tinyxml assimp ffmpeg
+	brew reinstall boost-python3 --without-python --with-python3 --build-from-source
+	conda install qt
+	export PKG_CONFIG_PATH=$(dirname $(dirname $(which python)))/lib/pkgconfig
+elif [[ "$(expr substr $(uname -s) 1 5)" == "Linux" ]]; then
+	sudo apt-get install -y ffmpeg pkg-config qtbase5-dev libqt5opengl5-dev libpython3.5-dev \
```

Comment:
To quote Angel again,

"It would be more readable to have each package per line"